streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -140,6 +140,16 @@ private[ui] class StreamingPage(parent: StreamingTab)
     <script>{Unparsed(js)}</script>
   }
 
+  private def generateTimeTipStrings(times: Seq[Long]): Seq[Node] = {
+    val js = "var timeTipStrings = {};\n" + times.map { time =>
+      val formattedTime =
+        SparkUIUtils.formatBatchTime(time, listener.batchDuration, showYYYYMMSS = false)
+      s"timeTipStrings[$time] = '$formattedTime';"
+    }.mkString("\n")
```

Comment:
This logic seems to be the same as [here](https://github.com/apache/spark/blob/master/streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala#L133).
Can we do like `s"timeTipStrings[$time] = timeFormat[$time];` or pull out the common code?